spec/models/manageiq/providers/kubernetes/container_manager/container_group_spec.rb

```diff
@@ -34,6 +34,7 @@
     persistent_volume = FactoryGirl.create(
       :persistent_volume,
       :name   => "persistent_volume",
+      :parent_type => "ManageIQ::Providers::ContainerManager",
```

Comment:
I believe your fix works both with correct parent_type ExtManagementSystem and this wrong parent_type.
While you're not fixing refresh on Euwe, it will keep creating only wrong ContainerManager parent_type (:+1: to that), please do test both.
That is, create 2 PVs here, call assert_volumes_relations twice, and similarly below.